python/res/enkf/key_manager.py

```diff
@@ -177,3 +177,15 @@ def isGenDataKey(self, key):
     def isMisfitKey(self, key):
         """ :rtype: bool """
         return key in self.misfitKeys()
+
+    def genKwPriors(self):
```

Comment:
I suggest we make this one camel case.

Reply:
In addition, this function is in dire need of a test or two ;)

Reply:
is it not the same format as all the other functions in this file?